### PR TITLE
AppOps rename

### PR DESCRIPTION
**Pull request by tristanlee85**

### CODE REVIEW

**Comment by RaeesBhatti** [comment]
@tristanlee85 I don't think AppOps would be a good replacement for Layer0 since we've divided it into two products "App Edge", "App Platform" and "App Security".
AppOps is a category name, not a product name. E.g. "This guide introduces the caching capabilities of AppOps" is not the intended meaning we want to convey.

**Comment by tristanlee85** [comment]
> @tristanlee85 I don't think AppOps would be a good replacement for Layer0 since we've divided it into two products "App Edge", "App Platform" and "App Security". AppOps is a category name, not a product name. E.g. "This guide introduces the caching capabilities of AppOps" is not the intended meaning we want to convey.

Agreed. This was a product name variable change, but we're having to go through and verify the content in which it was used. `Layer0` basically encompassed everything so `AppOps` isn't a good direct replacement.

File: constants.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const PRODUCT_NAME = 'Layer0';
+const PRODUCT_NAME = 'AppOps';
```
Comment (RaeesBhatti) [inline constants.js:1]: "Edgio" might be a good enough replacement here.

Comment (tristanlee85) [inline constants.js:1]:
```suggestion
const PRODUCT_NAME = 'Edgio';
```

**Comment by jsernaque** [comment]
@RaeesBhatti @tristanlee85 What do you think about adding constants for the company name and each product? This will allow us to immediately transition from Layer0 and allow us to iterate on the remaining instances.

```
const COMPANY_NAME = 'Edgio';
const PRODUCT_NAME = 'Edgio';
const PRODUCT_APPOPS_NAME = 'AppOps';
const PRODUCT_APPEDGE_NAME = 'App Edge';
const PRODUCT_APPPLATFORM_NAME = 'App Platform'
```

**Comment by jsernaque** [comment]
OK. The alternative is to strip out the variables where they don't fit and replace them with static values.

**Pull request merged**